descriptor.md

```diff
@@ -76,20 +76,17 @@ hex         := /[a-f0-9]+/
 
 Some example digest strings include the following:
 
-digest                                                                  | algorithm           |
+digest string                                                           | algorithm           |
```

Comment:
It's fine like this in context. If you want then submit a follow up that changes those AND the preceding two references.